packages/playground/assets/js/playground.js

```diff
@@ -0,0 +1,55 @@
+(function () {
+	const query = new URLSearchParams(window.location.search);
+	const defaultBlueprint = {
+		steps: [
+			{
+				step: 'login',
+			}
+		],
+	};
+
+	const blueprintUrl = query.get('blueprintUrl');
+	(async () => {
+		const { startPlaygroundWeb } = await import(
+			playground.playgroundPackageUrl
+		);
+		const blueprint = blueprintUrl
+			? await (await fetch(blueprintUrl)).json()
+			: defaultBlueprint;
+
+		blueprint.steps = blueprint.steps || [];
+		blueprint.steps = [
+			{
+				step: 'unzip',
+				zipFile: {
+					resource: 'url',
+					url: playground.zipUrl,
+				},
+				extractToPath: '/wordpress',
+			},
+			{
+				step: 'runSql',
```

Comment:
Lovely! I'd like Playground to do that automatically eventually, but today of course let's stick to an explicit step.